gpu-operator/release-notes.rst

```diff
@@ -33,6 +33,39 @@ Refer to the :ref:`GPU Operator Component Matrix` for a list of software compone
 
 ----
 
+
+
+.. _v25.10.1:
+
+25.10.1
+=======
+
+New Features
```

Comment:
We should call out https://github.com/NVIDIA/gpu-operator/pull/1894 as a new feature as well.